apis/v1alpha1/certificate.go

```diff
@@ -32,6 +33,7 @@ type CertificateSpec struct {
 	//
 	// arn:aws:acm-pca:region:account:certificate-authority/12345678-1234-1234-1234-123456789012
 	CertificateAuthorityARN *string `json:"certificateAuthorityARN,omitempty"`
```

Comment:
Similar to: https://github.com/aws-controllers-k8s/acmpca-controller/blob/main/apis/v1alpha1/certificate.go#L42

Can we allow the ref of a ACK PCA Certificate authority to be passed in?

Reply:
Makes sense. I'll look into it.

Reply:
I have made this change but this field was not introduced in this PR and is not related to importing certificates either.